manim/config.py

```diff
@@ -1,38 +1,37 @@
 """
 config.py
 ---------
-
 Process the manim.cfg file and the command line arguments into a single
 config object.
-
 """
-
 import os
 import sys
-import argparse
-import configparser
+
 import colour
-from .utils.tex import TexTemplateFromFile, TexTemplate
-from .logger import logger
+
 from . import constants
+from .utils.config_utils import _run_config, _init_dirs, _from_command_line
+
+from .logger import logger
+from .utils.tex import TexTemplate, TexTemplateFromFile
 
-__all__ = ["file_writer_config", "config", "camera_config"]
+__all__ = ["config", "camera_config"]
```

Comment:
question: why was `file_writer_config` removed? It needs to be exported as well.